io.github.Fontforge.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.Fontforge
```

Comment:
Has upstream expressed interest in (co-)maintaining the app on Flathub?

Reply:
I haven't asked yet. My plan was to propose the appstream file once it's good enough for flathub and ask the question then.

Reply:
I've asked in this PR: https://github.com/fontforge/fontforge/pull/3362